<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TwoDimWidget</class>
 <widget class="QWidget" name="TwoDimWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ID #0, Chan #0</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QwtPlot" name="mainPlot">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Display Settings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string/>
        </property>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QRadioButton" name="dispLin">
           <property name="text">
            <string>lin</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">linLogGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="dispLog">
           <property name="text">
            <string>log</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">linLogGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Module</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="moduleBox">
          <property name="maximum">
           <number>31</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Channel</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="channelBox">
          <property name="maximum">
           <number>127</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Mean:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="meanval">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Sigma:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="sigmaval">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Counts:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="countval">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Maximum:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxval">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>at Channel:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxpos">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_mouseOnChannel">
        <property name="text">
         <string>Channel 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>linLogGroup</sender>
   <signal>buttonClicked(int)</signal>
   <receiver>TwoDimWidget</receiver>
   <slot>displaychanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>channelBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TwoDimWidget</receiver>
   <slot>displaychanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1006</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>moduleBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TwoDimWidget</receiver>
   <slot>displaychanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1006</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>TwoDimWidget</receiver>
   <slot>clearHist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>955</x>
     <y>675</y>
    </hint>
    <hint type="destinationlabel">
     <x>1026</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>displaychanged()</slot>
  <slot>clearHist()</slot>
 </slots>
 <buttongroups>
  <buttongroup name="linLogGroup"/>
 </buttongroups>
</ui>
